src/Avalonia.Controls.DataGrid/Themes/Default.xaml

```diff
@@ -208,9 +208,9 @@
     <Setter Property="Template">
       <ControlTemplate>
         <Border BorderThickness="{TemplateBinding BorderThickness}" BorderBrush="{TemplateBinding BorderBrush}">
-          <Grid
-          RowDefinitions="Auto,*,Auto,Auto"
-          ColumnDefinitions="Auto,*,Auto">
+          <Grid Background="{TemplateBinding Background}"
```

Comment:
I think it might be better to apply the background to the `Border` where the `BorderThickness`/`BorderBrush` are defined, because if we were to add `CornerRadius` in future, there's an interaction between the border and the background that isn't handled well if they're defined on separate controls.

Unless there was a reason you applied it to the `Grid` and not the `Background`?